services/galley/src/Galley/Intra/Client.hs

```diff
@@ -18,11 +21,22 @@ import qualified Data.Set as Set
 
 lookupClients :: [UserId] -> Galley UserClients
 lookupClients uids = do
-    (h, p) <- brigReq
+    (brigHost, brigPort) <- brigReq
     r <- call "brig"
-        $ method POST . host h . port p
+        $ method POST . host brigHost . port brigPort
         . path "/i/clients"
         . json (UserSet $ Set.fromList uids)
         . expect2xx
     clients <- parseResponse (Error status502 "server-error") r
     return $ filterClients (not . Set.null) clients
+
+notifyClientsAboutLegalHoldRequest :: UserId -> UserId -> LastPrekey -> [Prekey] -> Galley ()
+notifyClientsAboutLegalHoldRequest requesterUid targetUid lastPrekey' prekeys = do
```

Comment:
take a `LegalHoldClientRequest` instead?  i may be missing something here, though.